support/ebpf/Makefile

```diff
@@ -44,16 +46,22 @@ OBJS := $(SRCS:.c=.$(TARGET_ARCH).o)
 
 .DEFAULT_GOAL := all
 
-all: $(TRACER_NAME)
+all: clean
+	$(MAKE) build
+	$(MAKE) clean
+	$(MAKE) TRACER_NAME=$(DEBUG_TRACER_NAME) debug
+	$(BPF_STRIP) -g $(DEBUG_TRACER_NAME)
```

Comment:
Not a fan of this: it forces a full rebuild every time. I think we should handle this the same way we do with `$(TARGET_ARCH)`: make debug/release part of the object file names so that Make automatically notices the changes. Also, since we're no longer committing the debug tracers into git, there's no need to strip them IMHO. The unstripped tracer results in much more readable verifier failures.

I pushed a commit (based on your changes) onto my fork to demonstrate this here: https://github.com/athre0z/opentelemetry-ebpf-profiler/commit/b7651fbbedd6d73fc1142a949e963a8ae85e61bc

If you agree that this is preferable, please feel free to just cherry-pick this into this PR.

Reply:
> Not a fan of this: it forces a full rebuild every time. I think we should handle this the same way we do with `$(TARGET_ARCH)`: make debug/release part of the object file names so that Make automatically notices the changes. Also, since we're no longer committing the debug tracers into git, there's no need to strip them IMHO. The unstripped tracer results in much more readable verifier failures.
> 
> I pushed a commit (based on your changes) onto my fork to demonstrate this here: [athre0z@b7651fb](https://github.com/athre0z/opentelemetry-ebpf-profiler/commit/b7651fbbedd6d73fc1142a949e963a8ae85e61bc)
> 
> If you agree that this is preferable, please feel free to just cherry-pick this into this PR.

Thanks. However, further modifications are needed to the `Makefile` in the project's root directory; otherwise, the file `ebpf/tracer.debug.ebpf.amd64` may not be found. I'll take care of it.